cli/src/factory_ops/unboxing/network/public_mgmt

```diff
@@ -109,6 +113,8 @@ function update_cluster_pillar_for_mgmt_ips {
         provisioner pillar_set --local cluster/srvnode-2/network/mgmt_nw/public_ip_addr \"${ip_2}\"
         provisioner pillar_set --local cluster/srvnode-2/network/mgmt_nw/gateway \"${gw_ip}\"
         provisioner pillar_set --local cluster/srvnode-2/network/mgmt_nw/netmask \"${netmask}\"
+        provisioner pillar_set --local cluster/srvnode-2/network/mgmt_nw/netmask \"${search_domain}\"
+        provisioner pillar_set --local cluster/srvnode-2/network/mgmt_nw/netmask \"${dns_server}\"
```

Comment:
the same as above:
- a typo (`netmask` is copied for both)
- json list string is expected here

Reply:
Addressed